joemam

```diff
@@ -0,0 +1,2 @@
+<h1> hi there </h1> 
+<p> who is joe </p>
```

Comment:
This feature is undocumented. Please include a docstring in the form of a HTML comment explaining the purpose of this tag.